src/common.rs

```diff
@@ -3387,6 +3387,75 @@ impl Cpu {
     pub fn frequency(&self) -> u64 {
         self.inner.frequency()
     }
+
+    /// Returns the CPU's arch (e.g. x86_64, arm64, ...).
+    ///
+    /// ```no_run
+    /// use sysinfo::{System, RefreshKind, CpuRefreshKind};
+    ///
+    /// let s = System::new_with_specifics(
+    ///     RefreshKind::new().with_cpu(CpuRefreshKind::everything()),
+    /// );
+    /// for cpu in s.cpus() {
+    ///     println!("{}", cpu.arch());
+    /// }
+    /// ```
+    pub fn arch(&self) -> CpuArch {
+        self.inner.arch()
+    }
+}
+
+/// Enumeration of CPU architectures
+#[derive(PartialEq, Clone, Debug)]
```

Comment:
You can derive `Copy` as well, that will allow to prevent calling `clone()`.

Reply:
You can also derive `Eq`.

Reply:
Fixed